packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -16,14 +19,31 @@ type PaymentConfirmationProps = {
   className?: string;
 };
 
+function getPlanPrice(interval: string, intervalCount: number, amount: string) {
+  switch (interval) {
+    case 'day':
+      if (intervalCount === 1) return `${amount} daily`;
+      return `${amount} every ${intervalCount} days`;
+    case 'week':
+      if (intervalCount === 1) return `${amount} weekly`;
+      return `${amount} every ${intervalCount} weeks`;
+    case 'month':
+      if (intervalCount === 1) return `${amount} monthly`;
+      return `${amount} every ${intervalCount} months`;
+    case 'year':
+      if (intervalCount === 1) return `${amount} yearly`;
+      return `${amount} every ${intervalCount} years`;
+  }
```

Comment:
Would be nice to restrict `interval` to the type `'day'|'week'|'month'|'year'`.  Otherwise the caller can pass in any other string and get an `undefined`.  Declaring a return type of `string` would also help.

Reply:
updated